Sources/Generator/UIGenerator.swift

```diff
@@ -105,7 +129,65 @@ public class UIGenerator: Generator {
                     }
                     try root.children.forEach { try generate(element: $0, superName: "target", containedIn: root) }
                     tempCounter = 1
-                    root.children.forEach { generateConstraints(element: $0, superName: "target") }
+                    root.children.forEach { generateConstraints(element: $0, superName: "target", forUpdate: false) }
+                    if configuration.isLiveEnabled {
+                        l("#endif")
+                    }
+                }
+                l()
+                l("func updateReactantUI()") {
+                    tempCounter = 1
+                    func generateChildVariables(children: [UIElement], guardNames: inout [String]) {
+                        for child in children {
+                            defer {
+                                if let container = child as? UIContainer {
+                                    generateChildVariables(children: container.children, guardNames: &guardNames)
+                                }
+                            }
+
+                            let name: String
+                            if child.field != nil {
```

Comment:
What do you think about a method that would collect names of the children? Very similar code is used in `generateChildProperties `.

Reply:
I think the code might have been blatantly copied and slightly modified, that's why they're so similar.